docs/installation.rst

```diff
@@ -1,19 +0,0 @@
-************
```

Comment:
sphinx with the auto section label decided including this file was an issue and wouldn't pull its teeth out of it despite being just fine before. It's not used anywhere else so I just inlined it with the index